### ISSUE TITLE
chore: use release-please instead of semantic-release

### ISSUE THREAD

**Issue by jthegedus** [jthegedus/svelte-adapter-firebase#105]
**Is your feature request related to a problem? Please describe.**

Use https://github.com/google-github-actions/release-please-action
 
**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by jthegedus** [comment]
Each of #110 #111 #113 #115 all failed to push to npm with the errors

```
404 Not Founds - registry.npm.js.org/svelte-adapter-firebase
```

**Comment by jthegedus** [comment]
Closed in #116 and released in #117
